lib/base/exception.cpp

```diff
@@ -112,10 +128,12 @@ void __cxa_throw(void *obj, TYPEINFO_TYPE *pvtinfo, void (*dest)(void *))
 
 	if (!uex) {
 #endif /* NO_CAST_EXCEPTION */
-		StackTrace stack;
+		// save the current stack trace in a thread-local variable
+		boost::stacktrace::stacktrace stack;
 		SetLastExceptionStack(stack);
 
 #ifndef NO_CAST_EXCEPTION
+		// save the current stack trace in the boost exception error info if the exception is a boost::exception
```

Comment:
A little picky, but I'd add a comma before the if.

Reply:
I think I learned in school that you don't place a comma there. :)

Reply:
You're right, my bad 😆 

https://dictionary.cambridge.org/de/grammatik/britisch-grammatik/conditionals